### DISCUSSION TITLE
Issue with connection to SatPC32

### DISCUSSION THREAD

**Discussion by geoham911** [magicbug/Cloudlog#1719] [Q&A]
Hi guys
Just upgrade to version 2 and having a problem with cat  data in the log.  May be I have missed something.
I have a Ubuntu server running on an intel NUC.  I installed and had version 1 of cloudlog working perfectly since last December. I regularly upgrade the server using apt-get etc

I upgrade to version 2 last night.  All working but log page not displaying the radio configuration. Mode or frequency or sat selected in SatPC32
Using SatPC32 with the cloudlog interface. Created a new read/write key and put it into the app. If I go into hardware interfaces i can see it is working ok but nothing coming through to the live logging page.

Have I missed something somewhere. I have downloaded and am testing the latest version of cloud interface but still have the issue

73
george
Gi4SJQ

![image](https://user-images.githubusercontent.com/24734884/197864281-b1ce1167-d226-421a-80b6-b5b8a864c1ba.png)

**Comment by magicbug** [comment]
Make sure your using the latest cloudlog to Satpc32 app, but also check in the admin - modes that usb & lsb are activated

**Reply by geoham911** [reply]
Hi Peter
Yes just checked that. I did a reboot or the server and logged back in.  Tracking RS-44 at the moment with the new version of the app.  showing up ok on the main page but nothing in the live log 
![image](https://user-images.githubusercontent.com/24734884/197876263-85872c67-c946-4ab2-a7db-9c732c2febc2.png)
Have to manually set the sat and mode. Checked and CW and SSB 
Will keep trying
George

**Reply by geoham911** [reply]
Just spotted my problem.
Have gone back through live logging page tabs and spotted that radio was no selected.  DA. Feed stupped now :-(

![image](https://user-images.githubusercontent.com/24734884/197877819-e235872f-7d5f-4edb-a430-31da6cf31555.png)

George
